tools/export.py

```diff
@@ -140,6 +148,33 @@ def main(args):
                         else (cfg.input_img_size, cfg.input_img_size))
     input_size = (3, input_time_size) + input_image_size
 
+    # BEGIN nncf part
+    was_model_compressed = is_checkpoint_nncf(args.checkpoint)
+    cfg_contains_nncf = cfg.get('nncf_config')
+
+    if cfg_contains_nncf and not was_model_compressed:
+        raise RuntimeError('Trying to make export with NNCF compression '
+                           'a model snapshot that was NOT trained with NNCF')
+
+    if was_model_compressed and not cfg_contains_nncf:
+        # reading NNCF config from checkpoint
+        nncf_part = get_nncf_config_from_meta(args.checkpoint)
+        for k, v, in nncf_part.items():
+            cfg[k] = v
+
+    if cfg.get('nncf_config'):
+        if torch.cuda.is_available():
+            model.cuda()
+        alt_ssd_export = getattr(args, 'alt_ssd_export', False)
```

Comment:
what is `alt_ssd_export`? it doesn't used anywhere.

Reply:
From mmdet version of export, will remove